pages/index/land-use.vue

```diff
@@ -136,26 +130,37 @@ export default {
         ...mapState('land-use', [
             'showFeatures',
             'features',
-            'table',
+            'tableLandUse',
             'visualizationStage',
+            'tableLand',
             'response',
             'params',
         ]),
     },
 
     methods: {
         search() {
-            this.getFeatures()
-            this.getDataTableLandUse()
+            if (this.tableLand === true) {
```

Comment:
Explain to me why is used validation with `===` instead of check variable value, like: `if(this.tableLand){...}`

Reply:
Resolved.